crates/utils/src/settings/structs.rs

```diff
@@ -65,7 +65,15 @@ pub struct PictrsConfig {
 
   /// Cache remote images
   #[default(true)]
-  pub cache_remote_images: bool,
+  pub cache_remote_thumbnails: bool,
```

Comment:
This setting might be unnecessary now. However the file request.rs where its used is quite the mess, not sure when I can get around to cleaning it up (it needs almost a complete rewrite).